vision/mnist/conv.jl

```diff
@@ -91,6 +101,11 @@ for epoch_idx in 1:100
     global best_acc, last_improvement
     # Train for a single epoch
     Flux.train!(loss, params(model), train_set, opt)
+    
+    if anynan(paramvec(model))
+        @error "NaN params"
+        break
+    end
```

Comment:
I would remove this check, it's not needed in an intro example and also is not something one commonly does in practice

Reply:
If we want to keep it, let's make the check more efficient:
```julia
anynan(m) = any(any(isnan, p) for p in params(m))
```

Reply:
I think I'm in favor of removing it too

Reply:
I get this error from time to time on my machine, when some of the parameters turn `NaN` and the accuracy then starts taking silly values and the model never converges. It could be related to some peculiar combination of CUDNN and GPU drivers that I have; would need to dig around a bit more to post a sensible bug report. I'd like to keep this check for now, to avoid saving models with NaN parameters.

Reply:
I understand but the purpose of this script is to serve as a basic template, so we should have here only standard stuff